core/src/main/java/org/apache/carbondata/core/datastore/filesystem/AbstractDFSCarbonFile.java

```diff
@@ -151,54 +149,47 @@ public String getPath() {
 
   @Override
   public long getSize() {
-    return fileStatus.getLen();
+    try {
+      return fileSystem.getFileStatus(path).getLen();
+    } catch (IOException e) {
+      throw new CarbonFileException("Unable to get file status: ", e);
+    }
   }
 
-  public boolean renameTo(String changeToName) {
-    FileSystem fs;
+  @Override
+  public boolean renameTo(String changetoName) {
     try {
-      if (null != fileStatus) {
-        fs = fileStatus.getPath().getFileSystem(hadoopConf);
-        return fs.rename(fileStatus.getPath(), new Path(changeToName));
-      }
+      return fileSystem.rename(path, new Path(changetoName));
```

Comment:
please add log for all file operation in this file. This class is the base class for other implementation, and we can have all info log in this class. Many times, we do not have sufficient logs to debug related to files operation issues